### PR TITLE
Adjusting AVM to NEF

### PR DESCRIPTION
**Pull request by vncoelho**
Adjusting https://github.com/neo-project/neo-devpack-dotnet/pull/55 which was renaming to NVM, according to neo-project/neo#903

### CODE REVIEW

**Comment by vncoelho** [comment]
Maybe after this PR we can refactor `NeoMethod` to `NefMethod`, right? @igormcoelho

**Comment by vncoelho** [comment]
@shargon, what NEF means? ahauahah

**Comment by shargon** [comment]
Neo Executable Format/File

File: tests/Neo.Compiler.MSIL.UnitTests/UnitTest1.cs [hunk 1]
```diff
@@ -8,18 +8,18 @@ namespace Neo.Compiler.MSIL
     [TestClass]
     public class UnitTest1
     {
-        private static void DumpAVM(NeoMethod avmMethod)
+        private static void DumpNEF(NeoMethod nefMethod)
```
Comment (erikzhang) [inline tests/Neo.Compiler.MSIL.UnitTests/UnitTest1.cs]: `neoMethod` is okay.

**Comment by shargon** [comment]
@lock9 I think that is done, could you tested?

File: tests/Neo.Compiler.MSIL.UnitTests/UnitTest1.cs [hunk 2]
```diff
@@ -8,18 +8,18 @@ namespace Neo.Compiler.MSIL
...
+            foreach (var c in nefMethod.body_Codes)
             {
                 Console.WriteLine(c.Key.ToString("X04") + "=>" + c.Value.ToString());
             }
         }
 
         private static void DumpBytes(byte[] data)
         {
-            Console.WriteLine("AVM=");
+            Console.WriteLine("NEF=");
```
Comment (igormcoelho) [inline tests/Neo.Compiler.MSIL.UnitTests/UnitTest1.cs]: Isn't this suppose to be NEF3 in little-endian? Isn't this the magic?

https://github.com/neo-project/neo/commit/f4adfac864e7844079120f50f2bee224aa3a406a#diff-e18ce5ffc990682ed3c619715ebaf672R27

**Pull request merged**